openwisp_radius/templates/djangosaml2/auth_error.html

```diff
@@ -0,0 +1,24 @@
+{% comment %}
+This template is taken from djangosaml2 with sole purpose of
+updating name space of the URL identifier.
+{% endcomment %}
+
+<!DOCTYPE html
+    PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+</head>
+
+<body>
+    <h1>Authorization error</h1>
+    <p>You are already logged in and you are trying to go to the login page again.</p>
+
+    <p>You may have been redirected here when trying to access some content
+        that required extra privileges that you do not have.</p>
+
+    <p>Please <a href="{% url 'radius:saml2_logout' %}">logout</a> and login as a different user</p>
```

Comment:
Let's flag strings and paragraphs as translatable using `trans` and `blocktrans`